helm/cluster-openstack/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: cluster-openstack
+description: A Helm chart for Kubernetes
```

Comment:
Nit but can we set the description here?

Reply:
I forgot about this. I've added a description now.